server/server_test.go

```diff
@@ -1684,6 +1684,35 @@ func (cli *testServerClient) runTestIssue3682(c *C) {
 	c.Assert(err.Error(), Equals, "Error 1045: Access denied for user 'issue3682'@'127.0.0.1' (using password: YES)")
 }
 
+func (cli *testServerClient) runTestEmptyRoles(c *C) {
```

Comment:
It's better to put the test into the same package, like expression/integration_test.go or expression/buildin_info_test.go

Reply:
ok. Let me have a try. I just add a test where `SELECT CURRENT_ROLE` occurs and it is here.

Reply:
`expression/integration_test.go` or `expression/buildin_info_test.go` doesn't test `CURRENT_ROLE` raw output, and here is the original test. I'd prefer adding tests as is and delay the test reorder later.

Reply:
I found that we do unit tests in these files. Will add more tests.

Reply:
done 636205971801ad8d3f0551c307c3626835d3419d